daemon/cmd/daemon_main.go

```diff
@@ -1436,6 +1437,21 @@ func (d *Daemon) instantiateAPI() *restapi.CiliumAPI {
 	return restAPI
 }
 
+func checkifKernelSockmapOptionsConfigured() {
+	if option.Config.SockopsEnable {
+		found := false
+		if h := probes.NewProbeManager().GetHelpers("sk_msg"); h != nil {
+			if _, ok := h["bpf_sock_hash_update"]; ok {
+				found = true
+			}
+		}
+		if !found {
+			log.Warn("BPF Sock ops not supported by kernel. Disabling the feature.")
```

Comment:
`Disabling '--sockops-enable'` like in original message to tell exactly what option failed?

Reply:
Sure will add it back to the message.